tests/runtime/call

```diff
@@ -825,3 +825,7 @@ AFTER ./testprogs/syscall nanosleep  1e8
 NAME block expression with map assignment
 PROG begin { @ = { let $a = 100; avg($a) };  }
 EXPECT @: 100
+
+NAME strlen
+PROG begin { $a = "hello"; print(strlen($a)); }
```

Comment:
Commit says "need to pass uint64 as the second arg" but the test passes just one arg. It's a bit confusing.